apps/gateway/package.json

```diff
@@ -1,33 +1,38 @@
 {
   "name": "@latitude-data/gateway",
-  "type": "module",
+  "version": "1.0.0",
+  "description": "Gateway for Latitude Data",
+  "private": true,
   "scripts": {
-    "build": "tsc --build tsconfig.prod.json --verbose",
+    "build": "tsup --config tsup.config.ts",
```

Comment:
`tsup` is a real bundler that allow us to transpile internal packages and mark the rest as externals